source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -46,7 +46,8 @@ namespace ExtAuthz {
   COUNTER(failure_mode_allowed)                                                                    \
   COUNTER(invalid)                                                                                 \
   COUNTER(ignored_dynamic_metadata)                                                                \
-  COUNTER(filter_state_name_collision)
+  COUNTER(filter_state_name_collision)                                                             \
+  COUNTER(omitted_response_headers)
```

Comment:
Document in `docs/root/configuration/listeners/network_filters/ext_authz_filter.rst`